src/xds.rs

```diff
@@ -76,11 +93,13 @@ impl ProxyStateUpdater {
     }
 
     pub fn insert_workload(&self, w: XdsWorkload) -> anyhow::Result<()> {
+        info!("handling insert {}", &w.uid);
```

Comment:
is this too noisy

Reply:
And with insert log, it will be clear whether a WL is updated or delete

Reply:
PR doesn't change the number of logs being generated at the info level. Prior to this PR for every insert we would log "handling delete {}" and with this PR we'll log "handing insert {}" instead.

If you change to debug log level then you will see additional logs because you'll also be seeing the delete logged for every insert. I thought being able to see the delete may be handy for debugging but would be too noisy normally.

Reply:
So can we tune both of then to debug or trace